### PR TITLE
Add PathoScope 2.0.6

### PR DESCRIPTION
**Pull request by davebx**
Addresses #323

### CODE REVIEW

File: packages/package_pathoscope_2_0_6/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <install version="1.0">
+            <actions>
+                <action type="setup_python_environment">
+                    <repository name="package_python_2_7_10" owner="iuc">
+                        <package name="python" version="2.7.10" />
+                    </repository>
+                    <package>https://github.com/PathoScope/PathoScope/archive/v2.0.6.zip</package>
+                </action>
+                <action type="set_environment">
+                    <environment_variable action="prepend_to" name="PYTHONPATH">$INSTALL_DIR</environment_variable>
```
Comment (yhoogstrate) [inline packages/package_pathoscope_2_0_6/tool_dependencies.xml]: Should the directory in which the pathoscope binary is located be prepended to $PATH?

Comment (davebx) [inline packages/package_pathoscope_2_0_6/tool_dependencies.xml]: @yhoogstrate Yes, but I need to find out why the binaries aren't being installed.

Comment (bgruening) [inline packages/package_pathoscope_2_0_6/tool_dependencies.xml]: Also PYTHONPATH is set automatically in Galaxy: https://github.com/galaxyproject/galaxy/blob/dev/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py#L1650

Comment (davebx) [inline packages/package_pathoscope_2_0_6/tool_dependencies.xml]: @bgruening So I discovered, upon inspecting the env.sh generated by the install process. Hold for updated xml.

**Comment by erasche** [comment]
@davebx you should be calling `setup.py`.

**Comment by erasche** [comment]
https://github.com/galaxyproject/tools-iuc/blob/master/packages/package_biopython_1_61/tool_dependencies.xml#L16

**Comment by davebx** [comment]
@erasche That's what setup_python_environment does.

**Comment by erasche** [comment]
you're darn right it does. My bad. LGTM.

**Comment by martenson** [comment]
thanks guys! ping @wanquan89

**Comment by erasche** [comment]
Not on TTS just yet. I'm waiting for the bot to do it. https://jenkins.galaxyproject.org/view/Tools/job/iuc-TTS-GH-update/1/

**Pull request merged**